docs/sources/auth/generic-oauth.md

```diff
@@ -80,6 +80,10 @@ Customize user login using `login_attribute_path` configuration option. Order of
 
 You can customize the attribute name used to extract the ID token from the returned OAuth token with the `id_token_attribute_name` option.
 
+You can set the user's display name with JMESPath using the `name_attribute_path` configuration option. It operates the same way as the `login_attribute_path` option.
+
+> **Note:** `name_attribute_path` is available in Grafana 7.4+.
```

Comment:
Why are we adding 7.4 docs before we have even finished 7.3?

Reply:
We want this feature in 7.4, not 7.3, because it's a feature and we already pushed 7.3 beta. So we don't want to merge this before 7.3 stable released.